modules/logging-upgrading-clo.adoc

```diff
@@ -22,12 +22,39 @@ To update the {clo} to a new major release version, you must modify the update c
 
 . Click the *Red Hat OpenShift Logging* Operator.
 
-. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.y*, depending on your current update channel.
+. Click *Subscription*. In the *Subscription details* section, click the *Update channel* link. This link text might be *stable* or *stable-5.<y>*, depending on your current update channel.
 
-. In the *Change Subscription Update Channel* window, select the latest major version update channel, *stable-5.y*, and click *Save*. Note the `cluster-logging.v5.y.z` version.
+. In the *Change Subscription Update Channel* window, select the latest major version update channel, *stable-5.<y>*, and click *Save*. Note the `cluster-logging.v5.<y>.<z>` version.
```

Comment:
fwiw: "major version" would mean 6.y (6.0 or 6.1 depending on your cluster version) at this point. The latest and last minor release of 5 is 5.9.